subsys/lorawan/lorawan.c

```diff
@@ -155,7 +155,7 @@ static void mcps_indication_handler(McpsIndication_t *mcps_indication)
 		if ((cb->port == LW_RECV_PORT_ANY) ||
 		    (cb->port == mcps_indication->Port)) {
 			cb->cb(mcps_indication->Port,
-			       !!mcps_indication->FramePending,
+			       mcps_indication->IsUplinkTxPending == 1,
```

Comment:
I blindly changed the name based on the following upstream change:
https://github.com/Lora-net/LoRaMac-node/commit/edff6ce161773e888687a9cd352ce2e2234e6b0d

However, I just got quite confused by the name. In my understanding, the `FramePending` indicated that there is another downlink message pending on the server and that we should send another uplink message to open a receive window for that pending message.

Here is the original discussion for that change:
https://github.com/Lora-net/LoRaMac-node/discussions/1142

I'll probably at least add a comment that it's actually a downlink rx message that is pending... or did I get something wrong?

Reply:
Added a comment to clarify the confusing name.